### ISSUE TITLE
Home page->Remove the button on 'No one' when profile is not shared 

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1118]
1) If a profile is not shared with anybody, then remove the click event from the 'My profile is shared with field' as it opens up a blank modal
This blank field and modal affects accessibility
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcUsyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--62cd5ed0988843447338de4dcf0cac1a1463a68e/Screenshot%202023-09-19%20at%203.43.07%20PM.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcU8yIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4883a7d03ea9d994090abd53b0584b9ee235cc3b/Screenshot%202023-09-19%20at%203.43.14%20PM.png)

2) If profile is shared with one user, then add the text 'Your profile is shared with "read the name". Click to view more details' to the button.  
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcjIyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--705eb44c3b953838d63688a5dff876bf1df14ad8/Screenshot%202023-09-19%20at%204.07.13%20PM.png)

3) If profile is shared with multiple users, then add the text 'Your profile is shared with "read the name" and 1 others. Click to view more details' to the button. 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcnkyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--726e195734391a8ddc63c42fcd47e0eb01176bec/Screenshot%202023-09-19%20at%204.06.43%20PM.png)

**Comment by steffipravasi** [comment]
1) Screen reader does not read the context in the modal on click more details for Shared Profile. 

2) Can we change the colour of the Close button to white and highlight it with border when hovered/focused
https://bcgov.sharepoint.com/:i:/r/teams/02915/Shared%20Documents/Product%20Team/Steffi%20-%20Testing/1118/Screenshot%202023-10-26%20at%203.30.56%20AM.png?csf=1&web=1&e=jf9qut

**Comment by steffipravasi** [comment]
@jizhaogit One minor issue here
For profile's shared with only one user, the text id displayed as _Name Click to view more details_
Can we add a full stop after the name? 
https://bcgov.sharepoint.com/:i:/r/teams/02915/Shared%20Documents/Product%20Team/Steffi%20-%20Testing/1118/Screenshot%202023-10-26%20at%2011.05.17%20AM.png?csf=1&web=1&e=TLexJn
For multiple shared profiles the text is _Name and 1 others. Click to view more details._ 
Can we match the text please?
https://bcgov.sharepoint.com/:i:/r/teams/02915/Shared%20Documents/Product%20Team/Steffi%20-%20Testing/1118/Screenshot%202023-10-26%20at%2011.06.21%20AM.png?csf=1&web=1&e=5rNjzE

**Comment by jizhaogit** [comment]
issue should be address.

**Comment by steffipravasi** [comment]
Test validated and issue has been addressed.
